rules-java-archives/victims-addon/src/main/java/org/jboss/windup/rules/victims/VictimsReportRules.java

```diff
@@ -0,0 +1,106 @@
+package org.jboss.windup.rules.victims;
+
+import org.jboss.windup.config.AbstractRuleProvider;
+import org.jboss.windup.config.GraphRewrite;
+import org.jboss.windup.config.metadata.RuleMetadata;
+import org.jboss.windup.config.operation.iteration.AbstractIterationOperation;
+import org.jboss.windup.config.phase.ReportGenerationPhase;
+import org.jboss.windup.config.query.Query;
+import org.jboss.windup.graph.GraphContext;
+import org.jboss.windup.graph.model.resource.FileModel;
+import org.jboss.windup.graph.model.ProjectModel;
+import org.jboss.windup.graph.model.WindupConfigurationModel;
+import org.jboss.windup.graph.service.GraphService;
+import org.jboss.windup.graph.service.WindupConfigurationService;
+import org.jboss.windup.rules.victims.model.AffectedJarModel;
+import org.jboss.windup.rules.victims.model.VictimsReportModel;
+import org.jboss.windup.reporting.model.TemplateType;
+import org.jboss.windup.reporting.service.ReportService;
+import org.jboss.windup.rules.apps.java.model.WindupJavaConfigurationModel;
+import org.ocpsoft.rewrite.config.ConditionBuilder;
+import org.ocpsoft.rewrite.config.Configuration;
+import org.ocpsoft.rewrite.config.ConfigurationBuilder;
+import org.ocpsoft.rewrite.context.EvaluationContext;
+
+
+/**
+ * Creates a report for all archives identified by the Victims service as containing a security vulnerability.
+ *
+ * @author Ondrej Zizka
+ */
+@RuleMetadata(tags = { "java" }, phase = ReportGenerationPhase.class)
+public class VictimsReportRules extends AbstractRuleProvider
+{
+    public static final String TITLE = "Security";
+    public static final String TEMPLATE_REPORT = "/org/jboss/windup/rules/victims/Report-Security.ftl.html";
```

Comment:
isn't that set by default? I sometime use Netbeans 8.1 and when I installed freemarker plugin it opens in HTML editor automatically and freemarker syntax highlighting and code completion works too.